daru.gemspec

```diff
@@ -51,6 +51,7 @@ EOF
 
   spec.add_runtime_dependency 'reportbuilder', '~> 1.4'
   spec.add_runtime_dependency 'spreadsheet', '~> 1.0.3'
+  spec.add_runtime_dependency 'dbi'
```

Comment:
OK the rest seems fine but this runtime dependency on dbi will involve additional dependencies on daru itself, which I'd like to keep to a minimal.

What you can do is add it as a development dependency, and use something like the `Daru.has_gsl?` method before doing a `require` on dbi. You can throw an error asking the user to do `gem install dbi` if he wants to use the `from_sql` method.

See this code to see how you can add a `Daru.has_dbi?` method: https://github.com/v0dro/daru/blob/master/lib/daru.rb#L57